File: 1150.py
def main():
    x, y = None, None
    x = int(input())
    
    while True:
        y = int(input())
        if y > x:
            break
    print(calc(x, y))


    
def calc(init: int, end: int) -> int:
    # 3 + 4 + 5 + 6 + 7
    count = 0
    sum = 0
    while True:
        sum += init
        count += 1
        if sum > end:
            return count
        init += 1

if __name__ == "__main__":
    main()

File: 1150.c
#include <stdio.h>

int calc(int init, int end) {
    int count = 0;
    int sum = 0;

    for(;;) {
        sum += init;
        count++;
        // printf("%d**", count);
        if (sum > end) {
            return count;
        }
        init++;
    }

}


int main() {
    int x;
    int y;
    scanf("%d", &x);
    do {
       scanf("%d", &y);
    } while (y <= x);
    // printf("PARAMS --> X %d ---> Y %d\n", x, y);
    
    int result = calc(x, y);
    printf("%d\n", result);
    return 0;
}